Alloy/commands/new/index.js

```diff
@@ -72,7 +80,7 @@ module.exports = function(args, program) {
 	);
```

Comment:
Just above this we're copying a `.gitignore` that needs to have `i18n` and `platform` added as well. The file copied is at https://github.com/cb1kenobi/alloy/blob/fe48f9a4b877e5933a216cc2cce72963cf1c6274/Alloy/template/gitignore.txt